service/etc/d-installer.yaml

```diff
@@ -7,3 +7,8 @@ software:
 distributions:
   # TODO replace simple id with better description of distributions
   - Tumbleweed
+
+web:
+  ssl: null
+  ssl_cert: null
+  ssl_key: null
```

Comment:
please document that new keys in markdown ( maybe I should add a note to file? )